src/osg_display/transfer_datasource.py

```diff
@@ -204,7 +203,10 @@ def query_transfers(self, starttime, endtime):
 
         response = gracc_query_transfers(self.es, transfers_raw_index, **params)
 
-        results = response.aggregations.StartTime.buckets
+        try:
+            results = response.aggregations.StartTime.buckets
+        except AttributeError:
```

Comment:
Interesting, does this trip over `AttributeError` sometimes?

Reply:
In testing, I ran across cases where the StartTime attribute didn't exist in the results, causing an exception. Perhaps time ranges where there were no data points? I didn't dig into the details, and the graph output looks comparable.